src/shared/common/forms/__tests__/dropdown.spec.js

```diff
@@ -9,6 +9,29 @@ const choices = [
     { label: 'isC', value: 'C' }
 ];
 
+// update to add active flags
+const conditionChoices = [
+    {
+        'conditionPropertyName': 'conditionValue',
+        'conditionPropertyValue': 'a',
+        'choices': [
+            { 'label': 'isA', 'value': 'A' },
+            { 'label': 'isB', 'value': 'B', 'active': true },
+            { 'label': 'isC', 'value': 'C', 'active': false },
+            { 'label': 'isD', 'value': 'D', 'active': false }
+        ]
+    },
+    {
+        'conditionPropertyName': 'conditionValue',
+        'conditionPropertyValue': 'b',
+        'choices': [
+            { 'label': 'isA', 'value': 'A' },
+            { 'label': 'isB', 'value': 'B', 'active': true },
+            { 'label': 'isC', 'value': 'C', 'active': false }
+        ]
+    },
+];
+
```

Comment:
Make it clearer what is being rendered in the tests by having the label include the condition value
```suggestion
const conditionChoices = [
    {
        'conditionPropertyName': 'conditionValue',
        'conditionPropertyValue': 'a',
        'choices': [
            { 'label': 'conditionA valueA', 'value': 'A' },
            { 'label': 'conditionA valueB', 'value': 'B', 'active': true },
            { 'label': 'conditionA valueC', 'value': 'C', 'active': false }
        ]
    },
    {
        'conditionPropertyName': 'conditionValue',
        'conditionPropertyValue': 'b',
        'choices': [
            { 'label': 'conditionB valueA', 'value': 'A' },
            { 'label': 'conditionB valueB', 'value': 'B', 'active': true },
            { 'label': 'conditionB valueC', 'value': 'C', 'active': false }
        ]
    },
];

```